contracts/modules/SuperMinterV1_1.sol

```diff
@@ -319,45 +320,58 @@ contract SuperMinterV1_1 is ISuperMinterV1_1, EIP712 {
 
         _incrementMinted(mode, d, p);
 
-        /* ----------- AFFILIATE AND PLATFORM FEES LOGIC ------------ */
+        /* ----------------- COMPUTE AND ACCRUE FEES ---------------- */
 
         TotalPriceAndFees memory f = _totalPriceAndFees(p.tier, d, p.quantity, p.signedPrice);
-
-        uint256 remaining; // The fee sent to the Sound Edition (i.e. artist fee).
-        bool affiliated;
+        MintedLogData memory l;
 
         unchecked {
             if (msg.value != f.total) revert WrongPayment(msg.value, f.total); // Require exact payment.
 
-            remaining = f.total - f.platformFee; // `platformFee <= total`;
-            platformFeesAccrued[d.platform] += f.platformFee; // Accrue the platform fee.
+            l.finalArtistFee = f.total - f.platformFee; // `platformFee <= total`;
+            l.finalPlatformFee = f.platformFee; // Initialize to the platform fee.
+            l.affiliate = p.to == p.affiliate ? address(0) : p.affiliate; // Yeah, we know it's left curved.
+
+            /* --------------------- AFFILIATE FEES --------------------- */
+
+            if (l.affiliated = _isAffiliatedWithProof(d, l.affiliate, p.affiliateProof)) {
+                // There are two kinds of affiliate fees:
+                // - The BPS based affiliateFee, which will be deducted from the artist's fee.
+                // - The per-mint flat affiliate incentive fee, which will be be deducted from the platform's fee.
 
-            if (affiliated = _isAffiliatedWithProof(d, p.affiliate, p.affiliateProof)) {
-                remaining -= f.affiliateFee; // `affiliateFee <= remaining`.
-                affiliateFeesAccrued[p.affiliate] += f.affiliateFee; // Accrue the affiliate fee.
+                // Deduct the BPS based affiliate fee from the artist's fee.
+                l.finalArtistFee -= f.affiliateFee;
+                // Deduct the affiliate incentive from the platform's fee.
+                l.finalPlatformFee -= f.affiliateIncentive;
```

Comment:
Tracking through all the calculations and configuration checks to try to resolve if this is underflowable gets too complex to be realiably accurate. Can we remove `unchecked` or do explicit checks for values that are especially prone to error (values that can't be easily commented on why it isn't underflow/overflowable).

Reply:
It is also worth considering that any underflow condition breaks balance tracking and will inevitably cause issues.

Reply:
Left it as checked math.

I think a few hundred gas more is worth the better assurance.

Most of our mints are on L2 anyways.